### PR TITLE
feat(Commands): Feedback CMD

### PR DESCRIPTION
**Pull request by Savage912-Official**
Ever wanted to provide feedback about the server to the staff team? You can do so by typing /feedback message in chat!

You can only use the command once per server restart.
Usage: /feedback message

Issue: #866 Completed

### CODE REVIEW

File: resources/usa_rp2/server.lua [hunk 1]
```diff
@@ -432,3 +432,50 @@ TriggerEvent('es:addCommand', 'discord', function(source, args, char)
...
+------------------------
+local cooldowns = {}
+TriggerEvent('es:addCommand','feedback', function(source, args, char, message)
+	local char = exports["usa-characters"]:GetCharacter(source)
+	local char_name = char.getFullName()
+	table.remove(args, 1)
+	local message = table.concat(args, ' ')
+	local msg = "**Server ID:** "..source.."\n**Steam Name:** "..GetPlayerName(source).."\n**Steam ID:** "..GetPlayerIdentifiers(source)[1].."\n**Character Name:** "..char_name.."\n**Character Playtime:** "..FormatSeconds(char.get("ingameTime")).." \n**Feedback:** "..message.."\n"
+	
+	if not cooldowns[source] or ((os.clock() - cooldowns[source]) > 86400) then
```
Comment (minipunch) [inline resources/usa_rp2/server.lua:447]: Where does the `86400` number come from ?

File: resources/usa_rp2/server.lua [hunk 2]
```diff
@@ -432,3 +432,50 @@ TriggerEvent('es:addCommand', 'discord', function(source, args, char)
...
+		cooldowns[source] = os.clock()
+		exports['erp_adminmenu']:sendToDiscord('New Feedback Submitted', msg, "16711680", GetConvar("feedback-webhook", ""))
```
Comment (minipunch) [inline resources/usa_rp2/server.lua:449]: I think we have an export for sending discord messages in `resources/[system]/globals` server script

File: resources/usa_rp2/server.lua (hunk 1 above)
Comment (minipunch) [reply resources/usa_rp2/server.lua:447]: Also if we are doing once per restart then you don't need to worry about using `os.clock` at all right

You can just add them to the `cooldowns` table and check if they have already been added or not

And the restart will reset the table

**Review by minipunch — CHANGES_REQUESTED** [review]

File: resources/usa_rp2/server.lua (hunk 1 above)
Comment (Savage912-Official) [reply resources/usa_rp2/server.lua:447]: > Where does the `86400` number come from ?

That was 24hr but working on updating it now

File: resources/usa_rp2/server.lua (hunk 2 above)
Comment (Savage912-Official) [reply resources/usa_rp2/server.lua:449]: > I think we have an export for sending discord messages in `resources/[system]/globals` server script

Working on this now

**Review by minipunch — APPROVED** [review]
Nice

**Pull request merged**